bin/versions

```diff
@@ -1,8 +1,8 @@
 MONO_VERSION=5.0.0.100
 Z3_SHORT_VERSION=4.8.1
 Z3_FULL_VERSION=4.8.1.016872a5e0f6
-BOOGIE_COMMIT=cd0609f660
-CORRAL_COMMIT=d7d389f22d
+BOOGIE_COMMIT=b694618b08
```

Comment:
I just merged your Boogie pull request and so you should update Boogie hash again to the latest one.